test/validation/api/ipsec/ipsec.c

```diff
@@ -288,6 +322,11 @@ void ipsec_sa_param_fill(odp_ipsec_sa_param_t *param,
 	if (in)
 		param->inbound.lookup_mode = ODP_IPSEC_LOOKUP_SPI;
 
+	if (in && (ODP_COS_INVALID != suite_context.cos)) {
+		param->inbound.pipeline = ODP_IPSEC_PIPELINE_CLS;
+		param->inbound.dest_cos = suite_context.cos;
+	}
```

Comment:
This test is going to need revision since the actual CoS is selected as a result of filtering the decrypted packet through PMRs that don't exist in this version of the test. Another reason to make pipeline support its own PR.

The test we'd want is

- Create a couple of CoS and associated queue objects.
- Create PMRs that filter packets into these CoSes.
- Create plaintext packets that will match the PMRs defined.
- Encrypt these packets via `odp_ipsec_out()`
- Now decrypt them via `odp_ipsec_in_enq()` with pipelining enabled and verify that the results show up on the expected queue based on the previously defined PMRs.